docs/modules/ROOT/partials/getting-started/con-registry-logging.adoc

```diff
@@ -0,0 +1,47 @@
+[id="registry-logging"]
+= {registry} Logging
+
+{registry} logging configuration can be modified while {registry} it's running. There is a REST endpoint that allows to configure the log level for specific loggers, allowing for finer grained logging configurations.
+
+[id="configuring-registry-logging"]
+== Configuring {registry} Logging during runtime
+
+This section explains how to check and configure log level in {registry} during runtime using the admin REST API.
+
+.Prerequisites
+
+* Obtain the URL to access your {registry} instance, get the route of your {registry} in case you have {registry} deployed on Openshift. In this guide we are using `localhost:8080`
```

Comment:
Suggest:
* Get the URL to access your {registry} instance, or get your {registry} route if you have {registry} deployed on OpenShift. This simple example uses `localhost:8080`.